src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -1263,6 +1263,11 @@ def _get_latest_revision(bucket_paths):
 
     build_urls.append(BuildUrls(bucket_path=bucket_path, urls_list=urls_list))
 
+  if len(build_urls) == 0:
+    logs.log_error("""Attempted to get latest revision,
```

Comment:
nit: Please use single quotes for strings. 

You can rely on implicit string continuation to break this over multiple lines.

(I haven't checked but it's possible you don't even need to break this over multiple lines if you just put the string
on the next line).

i.e.

````python
logs.log_error(
    'Attempted to get lateset revision,'
   'but not build urls were found.')
```